test/fixtures/point/canvas.js

```diff
@@ -1,6 +1,6 @@
 const canvas = window.createCanvas();
 module.exports = {
-  tolerance: 0.0055,
+  tolerance: 0.0095,
```

Comment:
How and why does it affect this fixture?

Reply:
![image](https://user-images.githubusercontent.com/11741250/147812603-9cbcf5cf-cc8a-40db-b8d6-e125d50ef838.png)

Reply:
I don't know why exactly but it seems the canvas provided as pointStyle works well, but there a difference on chartArea border.

Reply:
I was convinced to be related to chart area but now having a look better sounds related to the shadow, when pointStyle is a canvas

Reply:
My fault, apologize

Reply:
I forgot that pointStyle could be an image or canvas, where border is not applicable.